crypto/fipsmodule/rsa/rsa.c

```diff
@@ -742,24 +742,6 @@ static int check_mod_inverse(int *out_ok, const BIGNUM *a, const BIGNUM *ainv,
   return ret;
 }
 
-#if !defined(AWSLC_FIPS)
```

Comment:
Is FIPS okay with the relaxed check and d > n?

Reply:
that's a very good question. I believe it's fine because FIPS specifies requirements for key generation and our generation method ensures that `d` is in the correct range regardless of the removed check.